packages/mdc-fab/_mixins.scss

```diff
@@ -0,0 +1,154 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import "@material/animation/functions";
+@import "@material/animation/variables";
+@import "@material/elevation/mixins";
+@import "@material/ripple/mixins";
+@import "@material/theme/mixins";
+
+@function mdc-fab-theme-default-config_() {
+  @return (
+    standard: (
+      bg-color: secondary,
+      fg-color: text-primary-on-secondary,
+      ripple: (base-color: white, opacity: .16)
+    ),
+    plain: (
+      bg-color: white,
+      fg-color: text-primary-on-light,
+      ripple: ()
+    ),
+    disabled: (
+      bg-color: rgba(0, 0, 0, .12),
+      fg-color: rgba(0, 0, 0, .26),
+      ripple: null
+    ),
+  );
+}
+
+@function mdc-map-get-child_($parent-map, $keys...) {
+  $value: $parent-map;
+
+  @each $key in join($keys...) {
+    $value: map-get($value, $key);
+  }
+
+  @return $value;
+}
+
+@mixin mdc-fab-base {
+  @include mdc-ripple-base;
+
+  display: inline-flex;
+  position: relative;
+  justify-content: center;
+  width: 56px;
+  height: 56px;
+  padding: 0;
+  transition: box-shadow 280ms $mdc-animation-standard-curve-timing-function;
+  border: none;
+  border-radius: 50%;
```

Comment:
I believe border-radius (and possibly also border itself) may also end up being customizable. Maybe not necessary for a first pass but possibly worth noting.